forge-gui/res/cardsfolder/upcoming/cactus_preserve.txt

```diff
@@ -0,0 +1,9 @@
+Name:Cactus Preserve
+Types:Land Desert
+ManaCost:no cost
+K:CARDNAME enters the battlefield tapped.
+A:AB$ ManaReflected | Cost$ T | ColorOrType$ Type | Valid$ Land.YouCtrl | ReflectProperty$ Produce | SpellDescription$ Add one mana of any type that a land you control could produce.
+A:AB$ Animate | Cost$ 3 | Defined$ Self | Power$ X | Toughness$ X | Types$ Creature,Plant | Keywords$ Reach | SpellDescription$ Until end of turn, Cactus Preserve becomes an X/X green Plant creature with reach, where X is the greatest mana value among your commanders. It’s still a land.

```

Comment:
Missing `Colors$ Green`, `OverwriteColors$ True`, and `RemoveCreatureTypes$ True` here. 
The gist for the latter two is that you're setting colors and subtypes to a particular value without it being "in addition to its other [...]" which means this ability should overwrite what may already be there from other effects.

Reply:
This is good to know, thanks! Fixed.